### ISSUE TITLE
what is difference between RRDB_ESRGAN_x4 and RRDB_PSNR_x4

### ISSUE THREAD

**Issue by suke27** [xinntao/ESRGAN#85]

**Comment by gorinskaia** [comment]
As far as I understood, RRDB_ESRGAN_x4 is the current ESRGAN model, and RRDB_PSNR_x4 is a standart SRGAN one that gives smoother results (can be found here https://github.com/xinntao/BasicSR/blob/master/metrics/calculate_PSNR_SSIM.py).

**Comment by victorca25** [comment]
No, PSNR is the model trained only using content loss to initialize the filters. You use the PSNR model as a pre-trained model to train the ESRGAN model with feature loss and adversarial loss, so it converges faster to the global minima. Read the paper, it's explained there.
